core/src/main/java/org/apache/carbondata/core/cache/dictionary/ManageDictionaryAndBTree.java

```diff
@@ -131,22 +119,4 @@ public static void removeDictionaryColumnFromCache(AbsoluteTableIdentifier carbo
     dictCache = CacheProvider.getInstance().createCache(CacheType.FORWARD_DICTIONARY);
     dictCache.invalidate(dictionaryColumnUniqueIdentifier);
   }
-
-  /**
-   * This method will remove the BTree instances from LRU cache
-   *
-   * @param absoluteTableIdentifier
-   * @param segments
-   */
-  public static void invalidateBTreeCache(AbsoluteTableIdentifier absoluteTableIdentifier,
-      String[] segments) {
-    Cache<Object, Object> driverBTreeCache =
-        CacheProvider.getInstance().createCache(CacheType.DRIVER_BTREE);
```

Comment:
This code is required to clear Btree Cache of BLOCKLET_TREE

Reply:
handled.Updated the cachetype to blocklet

Reply:
blocket tree is handled through datamaps. not in this flow